src/components/Checkout/Sidebar/Sidebar.js

```diff
@@ -55,8 +46,27 @@ const Sidebar = () => {
           </FinalPrice>
         </Col>
       </Row>
+      {isCartPage && (
+        <Row>
+          <ButtonContainer>
+            <Button
+              onClick={() => {
+                if (typeof window !== "undefined") {
+                  window.location = "/checkout/customer";
```

Comment:
Instead of window.location you can use reach/router navigate api https://reach.tech/router/api/navigate

```suggestion
                navigate("/checkout/customer");
```

and: 
```js
import { navigate } from "@reach/router"
````
in import section